airbyte-workers/src/main/java/io/airbyte/workers/process/KubeProcessFactory.java

```diff
@@ -45,6 +45,7 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(KubeProcessFactory.class);
 
   private static final Pattern ALPHABETIC = Pattern.compile("[a-zA-Z]+");;
+  private static final String JOB_LABEL_KEY = "jobid";
```

Comment:
super nit: Does the formatting of this matter? Can it be `job_id`?

Reply:
yes!